docs/sources/setup/install/helm/install-scalable/aws.md

```diff
@@ -0,0 +1,580 @@
+  
+### EKS Minimum Requirements
+
+{{< admonition type="warning" >}}
+These EKS requirements are the minimum specification needed to deploy Loki using this guide. You may wish to adjust plugins and instance types based on your AWS environment and workload. **If you choose to do so, this may invalidate the guide.**
+
+In this guide, we deploy Loki using `m5.xlarge` instances. This is a middle-of-the-road instance type that should work for most scenarios. However, you can modify the instance types and count based on your specific needs.
+{{< /admonition >}}
+
+The minimum requirements for deploying Loki on EKS are: 
+
+- Kubernetes version `1.30` or above.
+- `3` nodes for the EKS cluster.
+- Instance type depends on your workload. A good starting point is `m5.xlarge`.
+
+Here is the EKSctl cluster configuration file used in this guide:
+
+```yaml
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: loki
+  region: eu-west-2
+  version: "1.31"
+
+iam:
+  withOIDC: true
+
+addons:
+  - name: aws-ebs-csi-driver
+  - name: eks-pod-identity-agent
+
+managedNodeGroups:
+  - name: loki-workers
+    instanceType: m5.xlarge
+    desiredCapacity: 3
+    minSize: 2
+    maxSize: 3
+    amiFamily: AmazonLinux2
+    iam:
+      withAddonPolicies:
+        ebs: true
+    volumeSize: 80
+    volumeType: gp2
+    ebsOptimized: true
+
+```
+
+
+The following plugins are also required to be installed within the EKS cluster:
+- Amazon EBS CSI Driver: Enables Kubernetes to dynamically provision and manage EBS volumes as persistent storage for applications. We use this to provision the node volumes for Loki.
+- Amazon EKS Pod Identity Agent: Manages AWS IAM roles for pods, allowing fine-grained access control to AWS resources without needing to store credentials in containers. This is how Loki will access the S3 bucket.
+- CoreDNS: Provides internal DNS service for Kubernetes clusters, ensuring that services and pods can communicate with each other using DNS names. 
+- kube-proxy: Maintains network rules on nodes, enabling communication between pods and services within the cluster.
+
+You must also install an OIDC provider on the EKS cluster. This is required for the IAM roles and policies to work correctly. If you are using EKSctl, you can install the OIDC provider using the following command:
+
+```bash
+eksctl utils associate-iam-oidc-provider --cluster loki --approve
+```
+**This may be installed by default when creating the EKS cluster using EKSctl.**
+
+## Create three S3 buckets
+
+Before deploying Loki, you need to create two S3 buckets; one to store logs (chunks), the second to store alert rules. You can create the bucket using the AWS Management Console or the AWS CLI. The bucket name must be globally unique. For this guide, we will use the bucket names `loki-aws-dev-chunks` and `loki-aws-dev-ruler` **but you should choose your own unique names when creating your own buckets**.
```

Comment:
do we own these buckets? if not we need to claim them, or switch these to buckets we do control. we should also make sure those bucket are in `deployment_tools` so we keep them as long as this documentation exists.

being extra cautious to prevent another https://grafana.com/blog/2024/06/27/grafana-security-update-grafana-loki-and-unintended-data-write-attempts-to-amazon-s3-buckets/

Reply:
Just working with @poyzannur on this. They currently sit in the devrel AWS account but we can get them moved over.